src/_includes/cookie-notice.html

```diff
@@ -0,0 +1,11 @@
+<section class="cookie-notice">
+  <div class="container">
+    <p>Google uses cookies to deliver its services, to personalize ads, and to 
+      analyze traffic. You can adjust your privacy controls anytime in your 
+      <a href="https://myaccount.google.com/data-and-personalization" target="_blank" rel="noopener" class="no-automatic-external">Google settings</a>. 
+      <a href="https://policies.google.com/technologies/cookies" target="_blank" rel="noopener" class="no-automatic-external">Learn more</a>.
+    </p>
+    <a id="cookie-consent" class="btn btn-primary"
+       tabindex="1" role="button">Okay</a>
```

Comment:
I believe this should be "0" to be focusable, but it still will not be usable by a keyboard user due to lacking an "href" attribute. 

Is it possible to use a normal `button` which will be inherently interactive.

Reply:
No problem. Button is great, that is what we have on flutter.dev. Changed it for docs, but you are right though. And you do have the compatible button styles so no issue on the dart side (changed, but not committed). 

A tabindex of 1 gives it first tab focus, and then the focus heads back to the header. The issue I am seeing is that Dartpad steals the focus permanently (in Chrome). I can also add this banner to the top of the html body so it's in a more ideal stacking order. I think it makes sense for it to get immediate focus though so it can be removed. 

Let me know! Thanks.

Reply:
I like the idea of it getting immediate focus, but it feels weird not being able to directly tab to the links with more information directly from it. That might get confusing.

I think either keep it last (so people can access it with shift-tab) or move the cookie notice to the top of the html body but keep it fixed to the bottom so all those links are at the beginning of the focus order together. Whichever you prefer is fine.

Reply:
I just tested on Flutter docs, moving the notice to the top of the `<body />` works great without setting a tabindex. That's much nicer for a screen reader as well.

Reply:
Sounds great to me!